app/stac_api/management/commands/remove_expired_items.py

```diff
@@ -0,0 +1,66 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.core.management.base import CommandParser
+from django.utils import timezone
+
+from stac_api.models import Item
+from stac_api.utils import CommandHandler
+from stac_api.utils import CustomBaseCommand
+
+
+class Handler(CommandHandler):
+
+    def delete(self, instance, object_type):
+        if self.options['dry_run']:
+            self.print_success(f'skipping deletion of {object_type} {instance}')
+        else:
+            instance.delete()
+
+    def run(self):
+        self.print_success('running command to remove expired items')
+        min_age_hours = self.options['min_age_hours']
+        self.print_warning(f"deleting all items expired longer than {min_age_hours} hours")
+        items = Item.objects.filter(
+            properties_expires__lte=timezone.now() - timedelta(hours=min_age_hours)
+        ).all()
+        for item in items:
+            assets = item.assets.all()
+            assets_length = len(assets)
+            self.delete(assets, 'assets')
+            self.delete(item, 'item')
+            if not self.options['dry_run']:
+                self.print_success(
+                    f"deleted item {item.name} and {assets_length}" + " assets belonging to it.",
+                    extra={"item": item.name}
+                )
+
+        if self.options['dry_run']:
+            self.print_success(f'[dry run] would have removed {len(items)} expired items')
```

Comment:
If I understand correctly this will end up as a normal log message as well?

Reply:
As discussed offline we will look at this separately.